webui/react/.gitignore

```diff
@@ -21,3 +21,6 @@ build-storybook/
 node_modules/
 
 report.*.json
+
+# .env with ESLINT_NO_DEV_ERRORS=true allows CRA to build with linting errors
+!.env
```

Comment:
what do you think about ignoring it or using a non react specific way to set the env vari?